env.mysql

```diff
@@ -7,3 +7,5 @@ DB_PORT=3306
 DB_DATABASE=lychee
 DB_USERNAME=root
 DB_PASSWORD=password
+CACHE_DRIVER=file
+LOG_VIEWER_CACHE_DRIVER=file
```

Comment:
Pretty sure these don't belong here. These are just for testing and are Docker env, not Lychee.

Reply:
They are actually also Lychee env.